gha/bin/release-init.py

```diff
@@ -235,6 +271,23 @@ def handle(self, base: str, head: str) -> NoReturn:
 
         self.log.info("Looks good")
 
+    def _get_commits_from_diff(self, sha1: str, sha2: str) -> list[Commit]:
+        self.log.info(f"Compare {sha1}..{sha2}")
+        return self.repo.compare(sha1, sha2).commits
+
+    def _parse_version_from_issue_summary(self, issue: Issue, repo_name: str) -> str:
+        release_version_re = (
+            RELEASE_VERSION_RE
+            if repo_name != "gbl-games-portal"
+            else RELEASE_VERSION_LEGACY_RE
+        )
+        self.log.info(f"Release version regex: {release_version_re.pattern}")
+        version_m = release_version_re.search(issue.fields.summary)
+        if not version_m:
+            raise AssertionError(
+                f"Cancelled release {issue.key} summary does not contain version")
```

Comment:
А уж здесь-то почему "Cancelled"? Эта функция вызывается дважды для разного типа тикетов.

```suggestion
                f"Release issue {issue.key} summary does not contain version")
```